requirements.txt

```diff
@@ -1,4 +1,5 @@
-azureml
-azureml-core
-click
-pyyaml
\ No newline at end of file
+azureml==0.2.7
```

Comment:
I'm a little bit scare about what is installed already on AML Studio VMs.  Have you tried that ?

Reply:
I'll take a look :)

Reply:
Hmm, I don't see a version of azureml installed on my Workspace VM.

![image](https://user-images.githubusercontent.com/17131668/117983840-393bfe80-b32f-11eb-8866-cdf7b1111ebf.png)

These are all the same versions of component libraries that seem to install with azureml==0.2.7 locally:

```bash
azureml                              0.2.7
azureml-automl-core                  1.27.0
azureml-core                         1.27.0
azureml-dataprep                     2.14.2
azureml-dataprep-native              33.0.0
azureml-dataprep-rslex               1.12.1
azureml-dataset-runtime              1.27.0
azureml-pipeline                     1.27.0
azureml-pipeline-core                1.27.0
azureml-pipeline-steps               1.27.0
azureml-telemetry                    1.27.0
azureml-train-automl-client          1.27.0
azureml-train-core                   1.27.0
azureml-train-restclients-hyperdrive 1.27.0
```

We can unpin this if you want.